_stylesheets/01-utilities/_utility.scss

```diff
@@ -27,7 +27,8 @@ img {
 	max-width: 100%;
 
 	&.full-width {
-		width: auto;
+    padding: $base-spacing 0;
```

Comment:
@jkaeser agree with above ^ ... can we just get a reformat of this whole file? Also, can probably remove [lines 1-13](https://github.com/ProboCI/probo-docs/blob/5f0b525ba82038d04b702b48902a6675a2973fa4/_stylesheets/01-utilities/_utility.scss#L1-L13)